File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio28.java
/*
Exercicio 028
Faça um Programa que pergunte em que turno você estuda.
Peça para digitar M-matutino ou V-Vespertino ou N- Noturno.

Imprima a mensagem "Bom Dia!",
"Boa Tarde!" ou "Boa Noite!"
 ou "Valor Inválido!", conforme o caso.
 */

import java.util.Scanner;
public class Exercicio28 {
    public static void main(String[] args){
    Scanner scanner = new Scanner(System.in);

    System.out.print("Digite 'M' se você estuda de manhã,'V' se você estuda de tarde e 'N' se você estuda de noite: " );
    String resposta = scanner.nextLine();

    if (resposta.equals("M")){
        System.out.print("Bom Dia! Você estuda de manhã");
    }
    else if(resposta.equals("V")){
        System.out.print("Boa Tarde! Você estuda de tarde");
    }
    else if (resposta.equals("N")){
        System.out.print("Boa Noite! Você estuda de noite");
    }
    else{
        System.out.print("Valor inválido!");
    }
    scanner.close();
}
}

File: Exercícios de Lógica/Python/Exercicio28.py
'''
Exercicio 028
Faça um Programa que pergunte em que turno você estuda. Peça para digitar M-matutino ou V-Vespertino ou N- Noturno.

Imprima a mensagem "Bom Dia!", "Boa Tarde!" ou "Boa Noite!" ou "Valor Inválido!", conforme o caso.

'''

turno = input("Digite o turno que você estuda (M,V,N): ").lower()

if turno == "m":
    print("Bom Dia!")

elif turno == "v":
    print("Boa Tarde!")

elif turno == "n":
    print("Boa noite!")

else:
    print("Valor Inválido!")